webui/src/styles/globals.css

```diff
@@ -300,6 +300,11 @@
   font-size: 0.85rem;
 }
 
+.unmarked-list {
```

Comment:
for classes with no semantic meaning that simply want to apply a simple style, you can use Bootstrap's spacing classes (see https://getbootstrap.com/docs/4.0/utilities/spacing/#examples)